public/app/features/library-panels/state/api.ts

```diff
@@ -76,9 +89,9 @@ export async function addLibraryPanel(
 }
 
 export async function updateLibraryPanel(panelSaveModel: PanelModelWithLibraryPanel): Promise<LibraryElementDTO> {
-  const { uid, name, version } = panelSaveModel.libraryPanel;
+  const { libraryPanel, ...model } = panelSaveModel;
```

Comment:
currently library panels are getting saved with the `libraryPanel` element embedded in them.  This removes it before saving